hypha/apply/users/middleware.py

```diff
@@ -13,3 +21,26 @@ def get_message(self, request, exception):
             return 'Your credentials are not recognised.'
 
         super().get_message(request, exception)
+
+
+class TwoFactorAuthenticationMiddleware:
```

Comment:
Let's add a doc string here:

```
Middleware to enforce 2FA activation for unverified users

To activate this middleware set env variable ENFORCE_TWO_FACTOR as True.

This will redirect all request from unverified users to enable 2FA first. Except the request made on the url paths listed in ALLOWED_SUBPATH_FOR_UNVERIFIED_USERS.
```